src/helpers/importedFiles.js

```diff
@@ -0,0 +1,47 @@
+'use strict';
+
+const path = require('path');
+
+const BemCell = require('@bem/sdk.cell');
+
+const cellPath = require('./cellPath');
+const namingConvention = require('./namingConvention');
+
+// TODO: suggest more elegant solution
+//   a. keeping full path in config
+//   b. allowing to build short bem-cell path without level part
+function resolveCellPath(relativePath, levelPath) {
+    return path.resolve(levelPath, '../' + relativePath);
+}
+
+/**
+ * Returns paths for BEM-entities across all levels
+ */
+module.exports = (entities, levels, { techs, techMap }) => {
+    let bemDeps = [];
+
+    entities.forEach(entity => {
+        const entityTechs = entity.tech ? [entity.tech] : techs;
+
+        entityTechs.forEach(tech => {
+            const techExt = techMap[tech] || tech;
+            const files = levels.map(level => {
+                const convention = namingConvention.getFromLevel(level);
+                const cell = BemCell.create({ entity, tech: techExt, layer: level.layer });
+
+                return {
+                    path: resolveCellPath(cellPath(cell, convention), level.path),
+                    tech
+                };
+            });
+
+            // TODO: should we add __elem (in case of __elem_mod) and _mod (in case of _mod_val) ?
```

Comment:
Yes

Reply:
Actually, not sure.

Reply:
@Yeti-or Can you tell us what actually `webpack-bem-loader` does?

Reply:
actually, it adds, why do you think it's unnecessary?

Reply:
[this part](https://github.com/bem/webpack-bem-loader/blob/master/index.js#L124-L129). I think we should simple remove this comment

Reply:
Can you add an example ?

Reply:
I've checked:

```js
parse('b:block e:elem m:mod')
/**
 * returns [
 *   { block: 'block', elem: 'elem' }, // __elem is here
 *   { block: 'block', elem: 'elem', mod: { name: 'mod' } }
 * ]
 */

parse('b:block m:mod=val')
/**
 * returns [
 *   { block: 'block' },
 *   { block: 'block', mod: { name: 'mod' } }, // _mod is here
 *   { lock: 'block', mod: { name: 'mod', val: 'val' } }
 * ]
 */
```

I'll remove nasty comment